doc/man3/OPENSSL_LH_stats.pod

```diff
@@ -46,13 +46,19 @@ are the same as the above, except that the output goes to a B<BIO>.
 
 These functions do not return values.
 
+=head1 NOTE
+
+These calls should be made under a read lock or inconsistent statistics could
+be output. Refer to L<OPENSSL_LH_COMPFUNC(3)> for more details about the locks
+required to use the LHASH data structure.
```

Comment:
This actually emphasizes my point quite vividly. It gives impression that read lock at OPENSSL_LH_stats is sufficient for consistent output. In this case would it be appropriate to omit "or inconsistent" and simply refer to OPENSSL_LH_COMPFUNC(3), and even more specifically to NOTE section?